### PR TITLE
792 spinner size

### PR DESCRIPTION
**Pull request by dexguitar**

### CODE REVIEW

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/spinner/GrowingSpinnersTests.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    @Test(dataProvider = "spinnerData")
+    public void spinnerHasGrowClassTest(Spinner spinner) {
+        spinner.highlight();
+        spinner.is().core().hasClass(spinnerGrowClass);
+    }
+
+    @Test(dataProvider = "spinnerData")
+    public void isValidationTest(Spinner spinner) {
+        spinner.highlight();
+        spinner.is().displayed().and().enabled();
```
Comment (mf-code) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/spinner/GrowingSpinnersTests.java]: put checks on different lines

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/spinner/SpinnerSizeTests.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        bsPage.shouldBeOpened();
+    }
+
+    @Test(dataProvider = "spinnerData")
+    public void isValidationTest(Spinner spinner) {
+        spinner.highlight();
+        spinner.is().enabled().and().displayed();
```
Comment (mf-code) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/spinner/SpinnerSizeTests.java]: put checks on different lines

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/spinner/GrowingSpinnersTests.java (hunk 1 above)
Comment (dexguitar) [reply jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/spinner/GrowingSpinnersTests.java]: Done

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/spinner/SpinnerSizeTests.java (hunk 2 above)
Comment (dexguitar) [reply jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/spinner/SpinnerSizeTests.java]: Done

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/spinner/GrowingSpinnersTests.java (hunk 1 above)
Comment (mf-code) [reply jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/spinner/GrowingSpinnersTests.java]: still actual

**Pull request merged**